db/migrations/20150717095310_fount.js

```diff
@@ -35,7 +35,7 @@ exports.up = function(knex, Promise) {
         table.string('url');
         table.integer('votes');
         table.integer('user_id').references('id').inTable('users');
-        table.timestamp('created_at');
+        table.timestamps();
```

Comment:
Unfortunately just changing this line here wont make the database change. We will have to write a new DB migration script that fixes this, as well as fixing the auto time stamp. I'm going to go ahead and approve the merge request, but I'm going to go ahead and raise a new issue on this since it will need to be fixed.